CRM/CiviAwards/Setup/AddAwardsCgExtendsOptionValue.php

```diff
@@ -19,6 +22,8 @@ public function apply() {
     ]);
 
     if ($result['count'] > 0) {
+      $this->toggle(TRUE);
```

Comment:
Can you take this line off? It's still referring to a non existent function also.